### DISCUSSION TITLE
Import playlists outside of MusicFolder

### DISCUSSION THREAD

**Discussion by rain0r** [navidrome/navidrome#2764] [Ideas]
It would be great if Navidrome could import playlists that are located outside of `MusicFolder`.

Suppose my `MusicFolder` is `/home/foo/Music`, then Navidrome is only able to import playlists that are located under `/home/foo/Music`, either top-level or nested, right?

In some cases it might not be possible to save the playlists in the `MusicFolder`, so it would be helpful to be able to give an absolute path to `PlaylistsPath`, e.g. `/home/foo/playlists`.

**Comment by dweymouth** [comment]
While it doesn't negate your feature request, it's worth noting that if you want your music and playlists in separate folders on your actual filesystem, you can still create a special root directory just for Navidrome that has symlinks to your Music and Playlists subdirectories.

This also works as a workaround for Navidrome lacking support for multiple root music directories
